### DISCUSSION TITLE
📣 IntelliJ users, please upgrade to version 1.1.20

### DISCUSSION THREAD

**Discussion by davecheney** [community/community#16696] [Copilot Conversations]
JetBrains pilots 👋 older versions of the copilot-intellij extension are no longer supported. Please upgrade to version 1.1.20 or later.

**Comment by Rain-Hreart** [comment]
May I ask why this problem appears on my side?  [ERROR] [default] [2022-05-25T2:14:53.151z] GitHub Copilot could not connect to server. Extension activation failed:  "unable to verify the first certificate"
Is there any solution?

**Reply by erindkushta** [reply]
Topis

**Comment by Pronphen** [comment]
https://github.com/github/feedback/discussions/16696#discussion-4076651
